feeadjuster/feeadjuster.py

```diff
@@ -225,13 +225,19 @@ def feeadjust(plugin: Plugin, scid: str = None):
 
     This method is automatically called in plugin init, or can be called manually after a successful payment.
     Otherwise, the plugin keeps the fees up-to-date.
+
+    To stop setting the channels with a list of nodes place a file called `feeadjuster-exclude.list` in the
+    lightningd data directory with a simple line-by-line list of pubkeys.
     """
     plugin.mutex.acquire(blocking=True)
     plugin.peers = plugin.rpc.listpeers()["peers"]
     if plugin.fee_strategy == get_fees_median and not plugin.listchannels_by_dst:
         plugin.channels = plugin.rpc.listchannels()['channels']
     channels_adjusted = 0
+    with open('feeadjuster-exclude.list') as file:
```

Comment:
1) The average user does not have a `feeadjuster-exclude.list` file. And without that, this line throws an exception:
`FileNotFoundError: [Errno 2] No such file or directory: 'feeadjuster-exclude.list'`

2) I'm not sure it will be obvious to everyone that the default folder for `feeadjuster-exclude.list` is `~/.lightning/bitcoin/`.

Reply:
good point, will post a change.